cpp/src/arrow/compute/kernels/minmax.cc

```diff
@@ -95,8 +95,12 @@ class MinMaxAggregateFunction final
       }
       reader.Next();
     }
-    *state = local;
 
+    if (array.null_count() > 0 && options_.skip_nulls == false) {
+      local.min = static_cast<typename ArrowType::c_type>(NAN);
```

Comment:
hi @bkietz, my intention is to return a null value here, however with different data type different values are returned:
- with float type, NAN is returned
- with integer type, 0 is returned   
this makes the unit tests code inconsistent. Is there a better way to return null here? Thanks!

Reply:
I would recommend extending `MinMaxState` with an explicit boolean field named `is_null` or so, then check that to determine if the resulting `Scalar`s should be null